pkg/miscellaneous/miscellaneous.go

```diff
@@ -338,6 +338,36 @@ func ProbeEtcd(ctx context.Context, clientFactory etcdClient.Factory, logger *lo
 	return nil
 }
 
+// IsMultiNode determines whether a pod is part of a multi node setup or not
+// This is determined by checking the `initial-cluster` of the etcd configmap to check the number of members expected.
+func IsMultiNode() bool {
+	var inputFileName string
+	etcdConfigForTest := os.Getenv("ETCD_CONF")
+	if etcdConfigForTest != "" {
+		return false
+	}
+	inputFileName = "/var/etcd/config/etcd.conf.yaml"
```

Comment:
I see this path being duplicated many times in the code. Can you create a constant and make the code use this?

Reply:
Done